### ISSUE TITLE
-s flag is unclear

### ISSUE THREAD

**Issue by bbushnell** [lrog/dsrc#15]
It's not obvious what the -s flag does.  There are basically 4 use cases for a program that does a transform on some data:

1) program infile outfile
2) program stdin outfile
3) program infile stdout
4) program stdin stdout

The definition of "-s" leaves it vague as to which of the last 3 is intended - so it's not clear how you differentiate between case 2 and case 3, or if case 4 is even supported.  Can you clarify how -s works in the compression and decompression modes?

**Comment by jvolkening** [comment]
I remember having this same thought when STDIN/STDOUT support was added, but you always need to give a filename for the <compressed> file being read from or written to. This makes some sense conceptually, since it's difficult to imagine a situation where you'd want/need to pipe the compressed bytes to or from another program. Agree that the documentation could be clearer on this, however.

**Comment by lrog** [comment]
Good point. This feature could have been explained more clearly. The only possible use-cases of using this feature are for reading and writing FASTQ data only, as there would be no big point on streaming in/out DSRC binary data in binary form. 
I've corrected the description, so hopefully now it should not make confusion:

> -s: read FASTQ data from stdin (compression) or write FASTQ data to stdout (decompression)
